Tests/pens/ttGlyphPointPen_test.py

```diff
@@ -0,0 +1,324 @@
+import os
+import unittest
+import struct
+
+from fontTools import ttLib
+from fontTools.misc.testTools import TestCase
+from fontTools.pens.ttGlyphPointPen import TTGlyphPointPen, MAX_F2DOT14
+from fontTools.ttLib.tables import ttProgram
+
+
+class TTGlyphPointPenTest(TestCase):
```

Comment:
for new test modules we should try to use the pytest API rather then unittest. Among the nice things it provides an easy way to parametrize repetitive tests. You probably copied form the existing ttGlyphPen_test.py which still uses unittest.

Reply:
Thanks, I changed the tests for both TTGlyphPen and TTGlyphPointPen to pytest.